tests/integration-tests/framework/credential_providers.py

```diff
@@ -58,7 +58,6 @@ def sts_credential_provider(region, credential_arn, credential_external_id=None,
 
     logging.info("Assuming STS credentials for region %s and role %s", region, credential_arn)
     aws_credentials = _retrieve_sts_credential(region, credential_arn, credential_external_id, credential_endpoint)
-    logging.info("Retrieved credentials %s", obfuscate_credentials(aws_credentials))
 
     try:
         logging.info("Unsetting current credentials %s", obfuscate_credentials(credentials_to_backup))
```

Comment:
it will then break here, when the STS retrieved credentials will be the current ones

Reply:
nope, because `_get_current_credentials` return a dict without the expiration:
```
    credentials_to_backup = _get_current_credentials()
...
def _get_current_credentials():
    return {
        "AWS_ACCESS_KEY_ID": os.environ.get("AWS_ACCESS_KEY_ID"),
        "AWS_SECRET_ACCESS_KEY": os.environ.get("AWS_SECRET_ACCESS_KEY"),
        "AWS_SESSION_TOKEN": os.environ.get("AWS_SESSION_TOKEN"),
        "AWS_PROFILE": os.environ.get("AWS_PROFILE"),
    }
```

Reply:
Modified the logic to avoid obfuscating the expiration field